### ISSUE TITLE
Vue routing

### ISSUE THREAD

**Issue by SauliusB** [dsuryd/dotNetify#135]
@dsuryd I am trying vue routing as per example. And I see my backend RoutingState is never been hit nor frontend onRouteEnter by links clicking.
I just wonder looking at react example I see 2 props at RouteLink - one for vm and one for link state and for vue example I see only vmRoute directive. Is it supposed to be that way? Thanks.

**Comment by dsuryd** [comment]
Yes, that’s correct.  Could you compare your setup with the code in Demo/Vue/SPA ?

**Comment by dsuryd** [comment]
Unless you're using `dotnetify.vue.$router.$mount` to mount the root component, make sure the Vue component has `RoutingState` in `data`.  You'll see console error if not.
